.vscode/settings.json

```diff
@@ -0,0 +1,21 @@
+{
+    "java.configuration.updateBuildConfiguration": "interactive",
+    "cmake.sourceDirectory": "${workspaceFolder}/TotalCrossVM",
+    "C_Cpp.default.configurationProvider": "ms-vscode.cmake-tools",
+    "files.associations": {
+        "__hash_table": "cpp",
+        "__split_buffer": "cpp",
+        "__tree": "cpp",
+        "atomic": "cpp",
+        "deque": "cpp",
+        "ios": "cpp",
+        "map": "cpp",
+        "set": "cpp",
+        "string": "cpp",
+        "system_error": "cpp",
+        "unordered_map": "cpp",
+        "vector": "cpp",
+        "__config": "cpp",
+        "__nullptr": "cpp"
+    }
+}
```

Comment:
I am not sure we should add editor settings to the repository. If so, I'd like to send my Emacs elisp files ;-)

Reply:
I also don't know if we should. @flsobral what do you think about it?

Reply:
if the settings don't automatically changes everytime you open or close the ide, and also, if it works for other machine without requiring changes, I think it's ok!

Reply:
removed